django/test/runner.py

```diff
@@ -518,8 +513,8 @@ def add_arguments(cls, parser):
             help='Runs a debugger (pdb, or ipdb if installed) on error or failure.'
         )
         parser.add_argument(
-            '-b', '--buffer', action='store_true',
-            help='Discard output from passing tests.',
+            '-nb', '--no-buffer', action='store_true', default=False,
+            help='outputs passing tests.',
```

Comment:
It would be clearer to the end-user if the help was "Shows output from passing tests."